sn_logging/src/metrics.rs

```diff
@@ -62,12 +56,12 @@ pub async fn init_metrics(pid: u32) {
             Some(safenode) => {
                 let disk_usage = safenode.disk_usage();
                 let process = ProcessMetrics {
-                    cpu_usage_percent: safenode.cpu_usage(),
-                    memory_used_mb: safenode.memory() as f32 / TO_MB,
+                    cpu_usage_percent: safenode.cpu_usage() as u32,
+                    memory_used_mb: safenode.memory() as u32 / TO_MB,
                     bytes_read: disk_usage.read_bytes,
                     bytes_written: disk_usage.written_bytes,
-                    total_mb_read: disk_usage.total_read_bytes as f32 / TO_MB,
-                    total_mb_written: disk_usage.total_written_bytes as f32 / TO_MB,
+                    total_mb_read: disk_usage.total_read_bytes as u32 / TO_MB,
+                    total_mb_written: disk_usage.total_written_bytes as u32 / TO_MB,
```

Comment:
As per the original types,

```rust
pub struct Process {
...
    pub(crate) memory: u64,
    pub(crate) virtual_memory: u64,
    cpu_usage: f32,
...
```

I think we should have memory in `u64` and CPU usage in `f32`